network/p2p/topic_validator_test.go

```diff
@@ -127,24 +127,26 @@ func TestTopicValidator(t *testing.T) {
 	wg.Wait()
 }
 
-func createID(t *testing.T) (*flow.Identity, crypto.PrivateKey) {
+func createID(t *testing.T, opts ...func(*flow.Identity)) (*flow.Identity, crypto.PrivateKey) {
 	networkKey, err := unittest.NetworkingKey()
 	require.NoError(t, err)
-	id := unittest.IdentityFixture(
-		unittest.WithRole(flow.RoleAccess),
-		unittest.WithNetworkingKey(networkKey.PublicKey()),
-	)
+	opts = append(opts, unittest.WithNetworkingKey(networkKey.PublicKey()))
+	id := unittest.IdentityFixture(opts...)
```

Comment:
So, it's completely non-obvious that this change is essential to the whole test component of the PR. I'd appreciate a one-liner comment on how this is used by non-AN non-topic-validator tests, or a move to a generic utils file, your choice.